src/views/Auth/components/ErrorMessage/index.js

```diff
@@ -1,36 +1,48 @@
 import React from 'react';
 import PropTypes from 'prop-types';
+import { Link } from 'react-router-dom';
+
 import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
 import { faExclamationTriangle } from '@fortawesome/pro-regular-svg-icons/faExclamationTriangle';
-
 import Box from '@material-ui/core/Box';
+import useStyles from './styles';
+
+const ErrorMessage = ({ message, link }) => {
+  const classes = useStyles();
 
-const ErrorMessage = ({ message }) => (
-  <Box
-    pl="10px"
-    pr="14px"
-    mb="-33px"
-    bottom={0}
-    border={1}
-    height={33}
-    display="flex"
-    color="#EF6A6E"
-    borderRadius={4}
-    bgcolor="#240F10"
-    alignSelf="center"
-    position="absolute"
-    alignItems="center"
-    borderColor="#EF6A6E"
-  >
-    <FontAwesomeIcon icon={faExclamationTriangle} />
-    <Box ml="7px" component="span" color="common.white">
-      {message}
+  return (
+    <Box className={classes.root}>
+      <FontAwesomeIcon icon={faExclamationTriangle} />
+      <Box className={classes.contentBox} component="span">
+        {message}
+        {
+          link !== undefined && (
+            <Box component="span">
+              <span>&#32;</span>
```

Comment:
Added a breaking space here, @giankotarola @gpuente

Reply:
@angarita-dev I think we can just replace the `<span>&#32;</span>` with `&nbsp;` html entity

Reply:
Added 👍 thanks for the code sample